config/configuration_file.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 Sonatype Inc.
```

Comment:
You could just name this `configuration.go`, in most cases we keep things fairly simple, and don't seperate things by "functionality" like how you do in Java. Golang values proximity (so large files with lots of stuff!) because it can reduce cognitive load (no jumping from file to file to see what's going on).